homeassistant/components/upnp/coordinator.py

```diff
@@ -35,12 +42,37 @@ def __init__(
             update_interval=update_interval,
         )
 
+    def register_entity(self, entity: "UpnpEntity") -> Callable[[], None]:
+        """Register an entity."""
+        # self._entities.append(entity)
+        key = entity.entity_description.key
+        entity_id = entity.entity_id
+        self._features_by_entity_id[key].add(entity_id)
```

Comment:
It would be a bit lower risk of breaking in the future if you pass `entity_id` and `key` so the there are no accesses to the entity object properties outside of itself.

Reply:
Fixed.